webapp/sources/rudder/rudder-web/src/main/resources/configuration.properties.sample

```diff
@@ -580,6 +580,40 @@ rudder.batch.reports.logInterval=1
 
 rudder.batch.storeAgentRunTimes.updateInterval=5
 
+##########################
+# Root server certificate ###########################################################
+##########################
+
+#
+# These parameters help configure what certificates are used for the root server.
+# They only set system properties that can be then used for the management, but
+# these parameters don't do anything more.
+#
+
+
+# Value for additional sever key hash
+# Will be used to trust more than one server certificate, typically for migration.
+# Await a semicolumn separated string array, on one line or with `\` at end of line because we are
+# in a properties file.
+# The given values will be appended to POLICY_SERVER_KEY_HASH variable in the curl compatible syntax.
+# rudder.server.certificate.additionalKeyHash=sha256//Pxjkq/Qlp02j8Q3ti3M1khEaUTL7Dxcz8sLOfGcg5rQ=;...
+# Default empty list.
+rudder.server.certificate.additionalKeyHash=
+
+#
+# Value for POLICY_SERVER_CERT_CA, ie the path to the file with the CA cert that should be used
+# for server certificate validation (the validation is not handled by Rudder).
+# Let empty for not restricting the list of possible CA to use.
+# Default empty.
+#
+rudder.server.certificate.ca.path=
+
+#
+# Value for POLICY_SERVER_SECURE_VALIDATION, ie should the server certificate
+# be checked. Default false or empty (ie `--insecure` passed to curl)
```

Comment:
this is confusing, and might make believe there is no validation when it is false

Reply:
so what would be not ?